bin/cqlsh.py

```diff
@@ -2188,6 +2205,28 @@ def read_options(cmdlineargs, environment):
     options.password = maybe_ensure_text(options.password)
     options.keyspace = maybe_ensure_text(options.keyspace)
 
+    serial_levels = [cassandra.ConsistencyLevel.SERIAL, cassandra.ConsistencyLevel.LOCAL_SERIAL]
```

Comment:
Think this should be a method extracted, but seems to be the style of this code to not do such things.